var/spack/repos/builtin/packages/cp2k/package.py

```diff
@@ -80,32 +80,57 @@ def install(self, spec, prefix):
             optflags = {
                 'gcc': ['-O2',
                         '-ffast-math',
-                        '-ffree-form',
-                        '-ffree-line-length-none',
                         '-ftree-vectorize',
                         '-funroll-loops',
                         '-mtune=native'],
                 'intel': ['-O2',
                           '-pc64',
                           '-unroll',
-                          '-heap-arrays 64']
+                          '-xHost']
```

Comment:
I'm not sure what to do either. Numpy also does something similar and @wscullin wasn't happy about it. I agree that `-xHost` is very useful if you are building for your own host, but if you aren't then I'm not sure if you can override it.